client/my-sites/pages/page.jsx

```diff
@@ -195,14 +195,14 @@ module.exports = React.createClass( {
 			return (
 				<PopoverMenuItem className="page__trash-item" onClick={ this.updateStatusTrash }>
```

Comment:
I see you replaced `page__trash-item` from style.scss, should this className be changed to `page__delete-item`?